src/components/common/button/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from "react";
+
+type ButtonProps = {
+  value: string;
+  buttonType: "button" | "submit";
+};
+
+const Button: FC<ButtonProps> = ({ value, buttonType }) => {
```

Comment:
I think `type` sounds better as a prop name, since the component is a `Button`

Reply:
True, I have adjusted it, thanks!